### PR TITLE
Preview subtitle clip in timeline

### PR DESCRIPTION
**Pull request by namse**
Subtitle clip would be not good for visibility so I renew it's layout.
During this changes, I updated camera clip's color and namui to support arc drawing.

# Subtitle Layout Renewal
image

Design doc : https://docs.google.com/document/d/1lXBEiRhDXzdGiQQTbLN3EWL3WGvCZazQ7v_tNzf10LI/edit#heading=h.vxdummyxmeri

# Camera Clip Color Renewal
image
I changed selected camera clip's background color.

### CODE REVIEW

**Comment by namse** [comment]
**Rebase** merge please

File: luda-editor/luda-editor-client/src/app/editor/timeline/timeline_body/track_body/subtitle_track_body/subtitle_clip_body.rs [hunk 1]
```diff
@@ -1,6 +1,8 @@
+use std::f32::consts::PI;
+
 use crate::app::{
     editor::{events::EditorEvent, TimelineRenderContext},
-    types::{PixelSize, SubtitleClip, Time},
+    types::{PixelSize, SubtitleClip, Time, TimePerPixel},
```
Comment (bigfoodK) [inline luda-editor/luda-editor-client/src/app/editor/timeline/timeline_body/track_body/subtitle_track_body/subtitle_clip_body.rs:5]:
```suggestion
    types::{PixelSize, SubtitleClip, Time},
```
`TimePerPixel` is unused. We can remove this

File: luda-editor/luda-editor-client/src/app/editor/timeline/timeline_body/track_body/subtitle_track_body/subtitle_clip_body.rs [hunk 2]
```diff
@@ -153,12 +130,251 @@ impl SubtitleClipBody {
...
                 check_flag,
-                tail_rendering_tree,
-                path(stroke_path, stroke_fill_paint),
             ],
         )
     }
 }
+
+struct SubtitleClipBodyRenderSetting {
+    clip_wh: Wh<f32>,
```
Comment (bigfoodK) [inline luda-editor/luda-editor-client/src/app/editor/timeline/timeline_body/track_body/subtitle_track_body/subtitle_clip_body.rs:159]:
```suggestion
```
`clip_wh` is unused. We can remove this

File: luda-editor/luda-editor-client/src/app/editor/timeline/timeline_body/track_body/subtitle_track_body/subtitle_clip_body.rs [hunk 3]
```diff
@@ -153,12 +130,251 @@ impl SubtitleClipBody {
...
+            top: xy.y,
+            right: xy.x + 1.0,
+            bottom: xy.y + 1.0,
+        }),
+        namui::PaintBuilder::new()
+            .set_color(color)
+            .set_style(PaintStyle::Fill),
+    )
+}
+
```
Comment (bigfoodK) [inline luda-editor/luda-editor-client/src/app/editor/timeline/timeline_body/track_body/subtitle_track_body/subtitle_clip_body.rs:182]:
```suggestion
```
This function is unused. Is it a function that should be reserved for debug? if not, we can remove this

**Review by bigfoodK — APPROVED** [review]

**Pull request merged**